corehq/pillows/mappings/reportcase_mapping.py

```diff
@@ -4,9 +4,9 @@
 from pillowtop.es_utils import ElasticsearchIndexInfo
 
 
-REPORT_CASE_INDEX = es_index("report_cases_czei39du507m9mmpqk3y01x72a3ux4p0")
+REPORT_CASE_INDEX = es_index("dm-case-attachment-refactor-8f7cba6")
```

Comment:
This index name should still be called `report_cases_<something>` so that it's clear which index it is. The convention with the other indexes is to append the date. You can always update the `comment` field if you want to add more details.